scripts/qa_crowbarsetup.sh

```diff
@@ -4586,6 +4586,14 @@ function onadmin_prepare_crowbar_upgrade()
         # move nodes to upgrade mode
         safely curl -s -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/prepare.json
     else
+        if crowbarctl upgrade prechecks --format plain | grep "false" ; then
+            complain 11 "Some necessary check before the upgrade has failed"
+        fi
```

Comment:
I agree it's not nice to require parsing in order to determine a binary result (success or failure). That's exactly what exit codes are supposed to be for, after all.